src/components/VDataTable/__snapshots__/VDataTable.spec.js.snap

```diff
@@ -87,7 +87,7 @@ exports[`VDataTable.vue should match a snapshot 1`] = `
                        style="display: inline-block;"
                   >
                     <div class="menu__content menu__content--select menu__content--auto"
-                         style="max-height: 200px; min-width: 75px; max-width: auto; top: 12px; left: 12px; z-index: 6; display: none;"
+                         style="max-height: 200px; min-width: 75px; max-width: auto; top: 12px; left: 12px; z-index: 0; display: none;"
```

Comment:
with `z-index: 0` - won't it hide behind the app toolbar for example?

Reply:
It pulls zero when not active.  Once active, it'll z-index in the higher.   I've committed a few more changes.